integration_tests/snaps/rust-musl/snapcraft.yaml

```diff
@@ -0,0 +1,14 @@
+name: rust-musl
```

Comment:
what would you think of using the basic rust yaml, and in the test setup append `libc: musl`, instead of adding a new snap?

Reply:
I guess now that you've also proposed new fixtures, this can be refactored once this lands.